source/sync/migrate-sync-modes.txt

```diff
@@ -285,6 +279,31 @@ start over from the beginning with future migration attempts.
    This will revert your migration. You will need to migrate again if you 
    want to enable Flexible Sync in the future.
 
+.. _realm-sync-migrate-after:
+
+After Backend Migration
+-----------------------
+
+Your migrated backend uses a partition key field in your Realm object models
+to map the Partition-Based Sync client objects to Flexible Sync equivalents on
+the backend. If your object models don't have a partition key field, the
+backend automatically injects one into every table the Partition-Based Sync
+client creates.
+
+The injected field follows this structure: ``<yourPartitionKey> == 
+<yourPartitionValue>``. Both the field key and value are derived from your 
+Partition-Based Sync client configuration.
```

Comment:
[nit] not sure if this is a distinction worth making, but technically `yourPartitionKey` is a backend parameter, while `yourPartitionValue` is a client config parameter

Reply:
Ah, thanks for clarifying.
> worth clarifying that the field name will be the partition key from their old backend PBS configuration, not the literal field name "partitionKey"

I guess I missed the "backend" part of what you said earlier. Sorry! I'll update.